<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student</class>
 <widget class="QDialog" name="student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生信息管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#student{
border-image: url(:/image/image/backpage_common.png);
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QPushButton" name="button_delete">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_delete{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_delete:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_delete:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#button_delete:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>姓名</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_class">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="placeholderText">
    <string>班级</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_revision">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_revision{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_revision:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_revision:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#button_revision:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_stu_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="placeholderText">
    <string>学号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_age">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/image/image/icon/arrow_down.png);
}


QComboBox#cbb_age{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_age:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>年龄</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbb_faculty">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/image/image/icon/arrow_down.png);
}


QComboBox#cbb_faculty{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_faculty:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地球科学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>资源学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>材料与化学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>环境学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地球物理与空间信息学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海洋学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>机械与电子信息学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>自动化学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>经济管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外国语学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地理与信息工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学与物理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>珠宝学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公共管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>计算机学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>体育学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>艺术与传媒学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>马克思主义学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>李四光学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高等教育研究所</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>远程与继续教育学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>国际教育学院</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_insert">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_insert{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_insert:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_insert:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#button_insert:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rbt_male">
      <property name="font">
       <font>
        <family>等距更纱黑体 SC</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>男</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sexGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbt_female">
      <property name="font">
       <font>
        <family>等距更纱黑体 SC</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>女</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sexGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="line_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>等距更纱黑体 SC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
   <property name="placeholderText">
    <string>身份证号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sexGroup"/>
 </buttongroups>
</ui>
